CHANGES.md

```diff
@@ -1,9 +1,20 @@
-=======================
  CHANGES for crate-pdo
 =======================
 
-Unreleased
-==========
+2018/03/08 1.0.0
```

Comment:
Please leave this as Unreleased, the header will become 1.0.0 once it's properly released (new branch, tag, etc)